src/profile/public/main.js

```diff
@@ -96,28 +93,6 @@ Promise.all([
 		}, '') : null
 	}
 
-	//
-	var words = [];
-
-	function resetPhrase() {
-		var arr = new Uint8Array(16);
-		window.crypto.getRandomValues(arr);
-		words = key_to_english(encode(arr)).split(' ');
-	}
-
-	$('.dropdown').on('show.bs.dropdown', function () {
-		resetPhrase();
-		var accum = '';
-		for (var i = 0; i < 4; i++) {
-			accum = accum + '<div class="form-row">';
-			for (var j = 0; j < 3; j++) {
-				accum = accum + '<div class="col-lg-4 form-group form-group-lg"><div class="form-control">' + words[3 * i + j] + '</div></div>';
-			}
-			accum = accum + '</div>';
-		}
-		$('.mnemonic-seed').html(accum);
-	});
-
```

Comment:
Again, we don't need to generate seed words thanks to the UserLib. In the future we will need a solution for recreating identity should it be lost or something, but we can cross that bridge when we come to it.